docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -34,44 +31,44 @@ All channels must integrate the following endpoints.
-| [Coupons](https://developer.bigcommerce.com/api-reference/store-management/marketing)                   | Manage coupons                                                            |
-| [Gift Certificates](https://developer.bigcommerce.com/api-reference/store-management/marketing)         | Manage gift certificates                                                  |
-| [Customer Login](https://developer.bigcommerce.com/api-docs/customers/customer-login-api)               | Use SSO to login customers                                                |
-| [Current Customer](https://developer.bigcommerce.com/api-docs/customers/current-customer-api)           | Securely identify current customer                                        |
-| [Order Payment Actions](https://developer.bigcommerce.com/api-reference/orders/orders-transactions-api) | Authorize, capture, and void order payments                               |
-| [Order Transactions](https://developer.bigcommerce.com/api-reference/orders/orders-transactions-api)    | Get order payment transaction data                                        |
+| Endpoint | Description |
+|-|-|
+|[Carts](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)|Create and manage carts|
+|[Checkouts](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-checkout-api)|Create and manage checkouts|
+|[Customers](https://developer.bigcommerce.com/api-reference/store-management/customers-v3)|Create and Manage Customers, Customer Addresses, and Customer Attributes.|
+|[Currencies](https://developer.bigcommerce.com/api-reference/store-management/currency-api)|Manage accepted currencies and their display|
+|[Coupons](https://developer.bigcommerce.com/api-reference/store-management/marketing)|Manage coupons|
+|[Gift Certificates](https://developer.bigcommerce.com/api-reference/store-management/marketing)|Manage gift certificates|
+|[Customer Login](https://developer.bigcommerce.com/api-docs/customers/customer-login-api)|Use SSO to login customers|
+|[Current Customer](https://developer.bigcommerce.com/api-docs/customers/current-customer-api)|Securely identify current customer|
+|[Order Payment Actions](https://developer.bigcommerce.com/api-reference/orders/orders-transactions-api)|Authorize, capture, and void order payments|
+|[Order Transactions](https://developer.bigcommerce.com/api-reference/orders/orders-transactions-api)|Get order payment transaction data|
 
 ## UI components
 
 To assist in the rapid development of apps that match the native UI and UX of the control panel, BigCommerce provides UI components and design guidelines to developers via [BigDesign](https://developer.bigcommerce.com/big-design/) -- BigCommerce’s library of React components.
 
-| Link                                                                                                                                 | Description                                            |
```

Comment:
I will defer to @aglensmith to comment on how the tables are formatted in this article.